pinot-connectors/pinot-spark-connector/src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister

```diff
@@ -0,0 +1 @@
+org.apache.pinot.connector.spark.datasource.PinotDataSourceV2
```

Comment:
what is V2?

Reply:
Spark has datasource read/write api v1 and v2. V2 api has support a lot of features like push down filters, columnar read etc. For for details [Spark DatasourceV2](https://www.slideshare.net/databricks/apache-spark-data-source-v2-with-wenchen-fan-and-gengliang-wang)